dnas/herd/zomes/coordinator/posts/src/comment.rs

```diff
@@ -179,19 +179,25 @@ pub fn downvote_comment(original_post_hash: ActionHash) -> ExternResult<()> {
 }
 
 #[hdk_extern]
-pub fn rmvote_comment(original_post_hash: ActionHash) -> ExternResult<()> {
-    create_link(
+pub fn rmvote_comment(comment_hash: ActionHash) -> ExternResult<()> {
+    get_links(
         agent_info()?.agent_initial_pubkey.clone(),
-        original_post_hash.clone(),
         LinkTypes::MyVotedComments,
-        (),
-    )?;
-    create_link(
-        original_post_hash,
-        agent_info()?.agent_initial_pubkey,
+        None,
+    )?.iter().for_each(|link| {
+        if link.target.clone().to_hex() == comment_hash.to_hex() {
+            let _ = delete_link(link.create_link_hash.clone());
```

Comment:
It is giving me the following error:
`{type: "error", data: {type: "internal_error", data: "Source chain error: InvalidCommit error: There are no link types in this integrity zome"}}`

Reply:
amazing thanks for raising this. I think you caught a bug in holochain core. I created a minimal example and raised in with the core team: https://github.com/holochain/holochain/issues/2541